src/ripple/overlay/README.md

```diff
@@ -147,41 +134,128 @@ Content-Type: application/json
 * `Connect-As`
 
     For requests the value consists of a comma delimited list of elements
-    where each element describes a possible connection type. Current connection
-    types are:
+    where each element describes a possible connection type. Only one
+    connection types is supported at present: **`peer`**.
 
-    - leaf
-    - peer
-
-    If this field is omitted or the value is the empty string, then 'leaf' is
-    assumed.
+    This field cannot be omitted.
 
     For responses, the value must consist of exactly one element from the list
     of elements specified in the request. If a server does not recognize any
     of the connection types it must return a HTTP error response.
 
+* `Network-ID` (optional)
+
+    If present, the `Network-ID` can be used to identify to which of several
+    [parallel networks](https://xrpl.org/parallel-networks.html) the sending
+    server is joined.
+
+    The value is a 32-bit unsigned integer. The following well-known values are
+    in use:
+
+    - **0**: The "main net"
+    - **1**: The Ripple-operated [Test Net](https://xrpl.org/xrp-test-net-faucet.html).
+
+* `Network-Time`
```

Comment:
It's optional. You can't really make it mandatory (it would break everyone!) and it can _only_ result in refused connections if both servers have a configured value and the values don't match.